### ISSUE TITLE
Cannot reassign comprand

### ISSUE THREAD

**Issue by mswider** [mswider/Monitor#2]
POST requests to [https://extapi.goguardian.com/api/v1/ext/nameandemail](https://extapi.goguardian.com/api/v1/ext/nameandemail), used to assign email addresses to comprands and switch users now requires a name, "url", and "Google profile id",  as well as an email address.  
Attempting to do such an assignment without _all_ of these, like how Monitor currently functions, fails with an http code 400.

The response looks something like this: 
```json
{
    "error": "name is required",
    "code": 3,
    "message": "name is required",
    "details": []
}
```
**Impact:** Worker comprands cannot be reassigned, meaning that things like chat updates will not work for users if you do not already have their comprand.

**Comment by mswider** [comment]
**Workaround:** You can manually make the request with the additional fields like below: 
Note that a name is now required, so doing this will probably change the display name associated with the email address across all GoGuardian services until the owner of said email logs on again and their extension changes it back.

```bash
curl --location --request POST 'https://extapi.goguardian.com/api/v1/ext/nameandemail' \
--header 'Authorization: COMPRAND' \
--header 'Content-Type: application/x-www-form-urlencoded' \
--data-urlencode 'email=EMAIL' \
--data-urlencode 'name=NAME' \
--data-urlencode 'googleProfileId=0' \
--data-urlencode 'url=chrome.identity'
````

You might notice that in this example, we're setting "googleProfileId" to 0, which turns out to be fine. The extension gets this from [chrome's identity API](https://developer.chrome.com/docs/extensions/reference/identity/#property-ProfileUserInfo-id), which I'm pretty sure returns a user's GAIA (Google Accounts and ID Administration ID). This is a unique ID for Google accounts, but I have no idea why GoGuardian needs this since their account system is based on the email address, and no behavior appeared to be out of the ordinary when setting it to a pointless value.

**Comment by mswider** [comment]
Closed in [v2.1.3](https://github.com/mswider/Monitor/releases/tag/v2.1.3)
